__tests__/lint/assertions-casing-camel-case-error/.redocly.yaml

```diff
@@ -4,20 +4,34 @@ apis:
 
 rules:
   assert/operation-get:
-    context:
-      - type: Operation
-        matchParentKeys: [get]
-    subject: Operation
-    property: operationId
+    where:
+      - subject: 
+          type: Operation
+          filterInParentKeys: [get]
+          property: responses
+        assertions:
+          defined: true
+    subject: 
+      type: Operation
```

Comment:
Can we use `filterInParentKeys` in this Operation directly without adding `where`?

Reply:
We can go with it. This is something that was overwritten during test refactoring